boards/FSM/Src/main.c

```diff
@@ -186,21 +186,21 @@ int main(void)
     SystemClock_Config();
 
     /* USER CODE BEGIN SysInit */
-
+    MX_DMA_Init(); // May not need!
     /* USER CODE END SysInit */
 
     /* Initialize all configured peripherals */
     MX_GPIO_Init();
-    MX_DMA_Init();
     MX_CAN_Init();
     MX_IWDG_Init();
     MX_ADC2_Init();
-    MX_TIM4_Init();
+    MX_DMA_Init();
```

Comment:
is this from cube version thats not 5.3.0? should probably update asap lol

Reply:
lol wtf i'm using 5.3 now :///

Reply:
i remeber you did something where you fully reload the auto-generated code. we can try that but can't remember how at the moment

Reply:
Yeah try doing a clean build

Reply:
make sure this is fixed after rebuilding

Reply:
Bump